index.html

```diff
@@ -12,8 +12,26 @@
 </head>
 
 <body>
+    <header>
```

Comment:
For your header, it is extending across the whole width of the page, you can change this and it will help you align it with the rest of the page.